event/event.gen.go

```diff
@@ -1029,20 +1029,6 @@ func TouchEnd(listener func(*vecty.Event)) *vecty.EventListener {
 	return &vecty.EventListener{Name: "touchend", Listener: listener}
 }
 
-// A touch point is moved onto the interactive area of an element.
-//
-// https://developer.mozilla.org/docs/Web/Events/touchenter
-func TouchEnter(listener func(*vecty.Event)) *vecty.EventListener {
-	return &vecty.EventListener{Name: "touchenter", Listener: listener}
-}
-
-// A touch point is moved off the interactive area of an element.
-//
-// https://developer.mozilla.org/docs/Web/Events/touchleave
-func TouchLeave(listener func(*vecty.Event)) *vecty.EventListener {
-	return &vecty.EventListener{Name: "touchleave", Listener: listener}
-}
-
```

Comment:
FYI these `TouchEnter` and `TouchLeave` events are removed because they were removed from the actual spec and were never really implemented.